### PR TITLE
Write documentation for lib/apachetest.pm

### PR DESCRIPTION
**Pull request by Zaoliang**
see https://progress.opensuse.org/issues/57062

### CODE REVIEW

File: lib/apachetest.pm [hunk 1]
```diff
@@ -23,6 +28,20 @@ our @EXPORT = qw(setup_apache2 setup_pgsqldb destroy_pgsqldb test_pgsql test_mys
...
+
+=head2 setup_apache2
+
+ setup_apache2();
+
+Setup Apache service in different mode: SSL, NSS, NSSFIPS, PHP7
+
+=over
+
+=item Example: setup_apache2(mode => 'SSL');
```
Comment (asdil12) [inline lib/apachetest.pm]: There is no reason for this to be a list.
Also the code should be in a code block (eg one empty line above and below and one space prefix).

File: lib/apachetest.pm [hunk 2]
```diff
@@ -23,6 +28,20 @@ our @EXPORT = qw(setup_apache2 setup_pgsqldb destroy_pgsqldb test_pgsql test_mys
 
 # Setup apache2 service in different mode: SSL, NSS, NSSFIPS, PHP7
 # Example: setup_apache2(mode => 'SSL');
+
+=head2 setup_apache2
+
+ setup_apache2();
```
Comment (asdil12) [inline lib/apachetest.pm]: Better like this:

```suggestion
 setup_apache2(mode => $mode);
```

File: lib/apachetest.pm [hunk 3]
```diff
@@ -23,6 +28,20 @@ our @EXPORT = qw(setup_apache2 setup_pgsqldb destroy_pgsqldb test_pgsql test_mys
...
+
+Setup Apache service in different mode: SSL, NSS, NSSFIPS, PHP7
```
Comment (asdil12) [inline lib/apachetest.pm]:
```suggestion
Setup Apache service in different mode.
Possivle values for C<$mode> are: SSL, NSS, NSSFIPS and PHP7
```

File: lib/apachetest.pm (hunk 2 above)
Comment (Zaoliang) [reply lib/apachetest.pm]: thanks!

**Pull request merged**